### PR TITLE
Add Portuguese translation

### PR DESCRIPTION
**Pull request by japakkjj**
§§a is the alternative color
§§m is the main color

### CODE REVIEW

**Comment by japakkjj** [comment]
@hampuskraft But is it right?

**Comment by hampuskraft** [comment]
Be patient. I'll review it when I've got time.

**Comment by japakkjj** [comment]
> Be patient. I'll review it when I've got time.

Ok

**Review by DidntPot — APPROVED** [review]

**Comment by japakkjj** [comment]
Nice

Enviado do meu iPhone

**Comment by japakkjj** [comment]
@hampuskraft  anything new?

File: portuguese.json [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    "formhandler.settings.pet.removed": "Pet removido.",
+    "formhandler.settings.pet.selected": "O seu pet de estimação foi mudado para um {%pet}.",
+    "formhandler.settings.showing": "exibindo",
+    "formhandler.switcher.doesntexist": "Uma conta com esse nome de usuário não existe! Por favor tente novamente.",
+    "forms.party.invite": "Convidar mais jogadores:",
+    "forms.reporter": "Reportar",
+    "forms.settings.fly": "Alternar voo",
+    "forms.settings.friends": "Permitir pedidos de amigos",
+    "forms.settings.hideguildtag": "Ocultar tag de guilda",
+    "forms.settings.hideplayers": "Ocultar Jogadores",
```
Comment (hampuskraft) [inline portuguese.json:59]: "Jogadores" shouldn't be capitalized here.

File: portuguese.json [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    "forms.settings.announcements": "Alternar anúncios do servidor",
+    "forms.settings.nick": "Altere o seu nick: (Use \"off\" ou \"reset\" para resetar para o padrão)",
+    "forms.settings.nick2": "Selecione um nickname:",
+    "forms.settings.pet": "Seu pet:",
+    "forms.settings.pet.change": "Mude o nome do seu pet: (Use \"off\" ou \"reset\" para resetar para o padrão)",
+    "forms.settings.pet.setname": "O nome do seu pet foi alterado para {%name}!",
+    "forms.teleporter": "Teleporter",
+    "forms.teleporter.full": "CHEIO",
+    "forms.teleporter.offline": "OFFLINE",
+    "forms.teleporter.partly_online": "PARTLY ONLINE",
```
Comment (hampuskraft) [inline portuguese.json:70]: Ironically, this seems to be partly translated—how about "EM PARTE ONLINE"?

**Review by hampuskraft — CHANGES_REQUESTED** [review]

File: portuguese.json [hunk 3]
```diff
@@ -0,0 +1,119 @@
+{
+  "name": "pt_br",
+  "pretty_name": "Portuguese",
+  "flag": "https://cdn.nethergames.org/img/flags/brazil.png",
+  "translators": ["LukXItado BR and Kingpotato1333"],
+  "translations": {
+    "global.nothere": "Não pode fazer isso aqui.",
```
[the remark this answers is not in the document]
Comment (hampuskraft) [reply portuguese.json]: Resolved in [`2f8c82b` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/2f8c82b3539fc568fdacb2d23c8da45451ef9526).

File: portuguese.json [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+    "secret.unlock.all": "Parabéns, completou os segredos do lobby! Que aventura selvagem foi esta. Aproveite o seu tempo, e esperamos que tenha gostado dos pequenos puzzles :)",
+    "zone.fly.disabled": "O modo de voo é desativado nesta zona",
+    "secret.unlocked.title": "Segredo desbloqueado",
+    "crate.inuse": "Esta caixa está sendo utilizada no momento",
+    "fireball.perfect.hit": "Você golpeou {%playerName} com {%blocks} blocos!",
+    "fireball.perfect.xp": "+{%xp} XP ({%perfectShot} tiro(s) perfeito(s) com fireball)",
+    "skywars.luckyblock.no_chests": "baús desativados no lucky block!",
+    "skywars.cooldown.item": "{%item} fica em tempo de espera durante {%seconds} segundos!",
+    "skywars.setup.add_middle_chest": "Adicionado com sucesso o peitoral médio em {%position}!",
+    "skywars.setup.enter_middle_chest_setup_mode": "Modo de configuração do baú para: {%arena}\nTap baús para adicionar ao baú central ou usar comando '/sw editar set do baú de id para parar.",
```
Comment (hampuskraft) [inline portuguese.json:103]: This translation is incorrect. The original string is "Entered chest configuration mode for: {%arena}\nTap chests to add to middle chests or use command '/sw edit setmidchest' to stop."

**Comment by hampuskraft** [comment]
@japakkjj As you can see, I've added plenty of comments. Please resolve these before we can merge your PR. Let me know if you have any questions. Otherwise, Google is your friend.

File: portuguese.json (hunk 2 above)
Comment (hampuskraft) [reply portuguese.json]: Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).

File: portuguese.json [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+    "formhandler.reporter.reported": "Você reportou {%reported} por {%reason}",
+    "formhandler.reporter.staffmessage": "{%reported} foi reportado {%reporter} por {%reason}",
+    "formhandler.settings.disabled": "Desativado",
+    "formhandler.settings.enabled": "Ativado",
+    "formhandler.settings.fly": "{%enabledOrDisabled} voar no lobby.",
+    "formhandler.settings.fly.hidingrank": "Você não pode voar enquanto estiver usando um nome falso ou a esconder a sua posição de rank.",
+    "formhandler.settings.friends": "Agora vocês {%enabledOrDisabled} São amigos.",
+    "formhandler.settings.guildtag": "Agora é {%enabledOrDisabled} your guild tag.",
+    "formhandler.settings.hideplayers": "You are now {%enabledOrDisabled} players.",
+    "formhandler.settings.hiding": "escondido",
```
[the remark this answers is not in the document]
Comment (hampuskraft) [reply portuguese.json]: Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).

File: portuguese.json [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+    "db.error": "Ocorreu um erro inesperado na conexão com o banco de dados. Por favor, tente novamente mais tarde.",
+    "formhandler.reporter.notreported": "Esse jogador não pode ser reportado.",
+    "formhandler.reporter.reported": "Você reportou {%reported} por {%reason}",
+    "formhandler.reporter.staffmessage": "{%reported} foi reportado {%reporter} por {%reason}",
+    "formhandler.settings.disabled": "Desativado",
+    "formhandler.settings.enabled": "Ativado",
+    "formhandler.settings.fly": "{%enabledOrDisabled} voar no lobby.",
+    "formhandler.settings.fly.hidingrank": "Você não pode voar enquanto estiver usando um nome falso ou a esconder a sua posição de rank.",
+    "formhandler.settings.friends": "Agora vocês {%enabledOrDisabled} São amigos.",
+    "formhandler.settings.guildtag": "Agora é {%enabledOrDisabled} your guild tag.",
```
[the remark this answers is not in the document]
Comment (hampuskraft) [reply portuguese.json]: Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).

File: portuguese.json [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+    "command.tell.receiver": "{%sender} sussurrado para: {%message}",
+    "command.tell.sender": "O senhor sussurrou para {%receiver}: {%message}",
+    "command.tell.yourself": "você não pode enviar uma mensagem a si mesmo.",
+    "command.tp.specify": "Você deve especificar um jogador.",
+    "command.tphere.accepted.receiver": "Aceite o pedido de {%sender} para se teleportarem para eles.",
+    "command.tphere.accepted.sender": "{%receiver} aceitou o seu pedido para que eles se teletransportem para você.",
+    "command.tphere.declined.receiver": "recusou o pedido de {%sender} para que se teleportasse para eles.",
+    "command.tphere.declined.sender": "{%receiver} recusou o seu pedido para que eles se teleportassem para você.",
+    "command.tphere.noperm": "Não tem permissão para pedir a outros jogadores para se teletransportarem para você. Adquira o passe §l§aEMERALD§r §cor §l§bLEGEND§r §§mrank em §§abngmc.co/store §§mno site!",
+    "command.tphere.norequest": "Aquele jogador não enviou um pedido para que se teletransporte para eles.",
```
[the remark this answers is not in the document]
Comment (hampuskraft) [reply portuguese.json]: Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).

File: portuguese.json [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+    "command.ci.completed": "O seu inventário foi limpo!",
+    "command.notinlobby": "Esse comando só pode ser usado no lobby.",
+    "command.tell.receiver": "{%sender} sussurrado para: {%message}",
+    "command.tell.sender": "O senhor sussurrou para {%receiver}: {%message}",
+    "command.tell.yourself": "você não pode enviar uma mensagem a si mesmo.",
+    "command.tp.specify": "Você deve especificar um jogador.",
+    "command.tphere.accepted.receiver": "Aceite o pedido de {%sender} para se teleportarem para eles.",
+    "command.tphere.accepted.sender": "{%receiver} aceitou o seu pedido para que eles se teletransportem para você.",
+    "command.tphere.declined.receiver": "recusou o pedido de {%sender} para que se teleportasse para eles.",
+    "command.tphere.declined.sender": "{%receiver} recusou o seu pedido para que eles se teleportassem para você.",
```
[the remark this answers is not in the document]
Comment (hampuskraft) [reply portuguese.json]: Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).

File: portuguese.json [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+  "translations": {
+    "global.nothere": "Não pode fazer isso aqui.",
+    "command.nothere": "Não pode usar esse comando aqui!",
+    "command.ci.completed": "O seu inventário foi limpo!",
+    "command.notinlobby": "Esse comando só pode ser usado no lobby.",
+    "command.tell.receiver": "{%sender} sussurrado para: {%message}",
+    "command.tell.sender": "O senhor sussurrou para {%receiver}: {%message}",
+    "command.tell.yourself": "você não pode enviar uma mensagem a si mesmo.",
+    "command.tp.specify": "Você deve especificar um jogador.",
+    "command.tphere.accepted.receiver": "Aceite o pedido de {%sender} para se teleportarem para eles.",
```
[the remark this answers is not in the document]
Comment (hampuskraft) [reply portuguese.json]: Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).

File: portuguese.json [hunk 10]
```diff
@@ -0,0 +1,119 @@
...
+    "command.vote.slowdown": "Calma, já estamos verificando os votos para você!",
+    "db.error": "Ocorreu um erro inesperado na conexão com o banco de dados. Por favor, tente novamente mais tarde.",
+    "formhandler.reporter.notreported": "Esse jogador não pode ser reportado.",
+    "formhandler.reporter.reported": "Você reportou {%reported} por {%reason}",
+    "formhandler.reporter.staffmessage": "{%reported} foi reportado {%reporter} por {%reason}",
+    "formhandler.settings.disabled": "Desativado",
+    "formhandler.settings.enabled": "Ativado",
+    "formhandler.settings.fly": "{%enabledOrDisabled} voar no lobby.",
+    "formhandler.settings.fly.hidingrank": "Você não pode voar enquanto estiver usando um nome falso ou a esconder a sua posição de rank.",
+    "formhandler.settings.friends": "Agora vocês {%enabledOrDisabled} São amigos.",
```
[the remark this answers is not in the document]
Comment (hampuskraft) [reply portuguese.json]: Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).

File: portuguese.json [hunk 11]
```diff
@@ -0,0 +1,119 @@
...
+  "pretty_name": "Portuguese",
+  "flag": "https://cdn.nethergames.org/img/flags/brazil.png",
+  "translators": ["LukXItado BR and Kingpotato1333"],
+  "translations": {
+    "global.nothere": "Não pode fazer isso aqui.",
+    "command.nothere": "Não pode usar esse comando aqui!",
+    "command.ci.completed": "O seu inventário foi limpo!",
+    "command.notinlobby": "Esse comando só pode ser usado no lobby.",
+    "command.tell.receiver": "{%sender} sussurrado para: {%message}",
+    "command.tell.sender": "O senhor sussurrou para {%receiver}: {%message}",
```
[the remark this answers is not in the document]
Comment (hampuskraft) [reply portuguese.json]: Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).

File: portuguese.json [hunk 12]
```diff
@@ -0,0 +1,119 @@
...
+    "command.tell.yourself": "você não pode enviar uma mensagem a si mesmo.",
```
[the remark this answers is not in the document]
Comment (hampuskraft) [reply portuguese.json]: Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).

File: portuguese.json (hunk 9 above)
[the remark this answers is not in the document]
Comment (hampuskraft) [reply portuguese.json]: Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).

File: portuguese.json [hunk 13]
```diff
@@ -0,0 +1,119 @@
...
+    "command.tp.specify": "Você deve especificar um jogador.",
+    "command.tphere.accepted.receiver": "Aceite o pedido de {%sender} para se teleportarem para eles.",
+    "command.tphere.accepted.sender": "{%receiver} aceitou o seu pedido para que eles se teletransportem para você.",
```
[the remark this answers is not in the document]
Comment (hampuskraft) [reply portuguese.json]: Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).

File: portuguese.json [hunk 14]
```diff
@@ -0,0 +1,119 @@
...
+    "command.tphere.declined.receiver": "recusou o pedido de {%sender} para que se teleportasse para eles.",
```
[the remark this answers is not in the document]
Comment (hampuskraft) [reply portuguese.json]: Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).

File: portuguese.json (hunk 8 above)
[the remark this answers is not in the document]
Comment (hampuskraft) [reply portuguese.json]: Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).

File: portuguese.json (hunk 7 above)
[the remark this answers is not in the document]
Comment (hampuskraft) [reply portuguese.json]: Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).

File: portuguese.json [hunk 15]
```diff
@@ -0,0 +1,119 @@
...
+    "command.tphere.declined.sender": "{%receiver} recusou o seu pedido para que eles se teleportassem para você.",
+    "command.tphere.noperm": "Não tem permissão para pedir a outros jogadores para se teletransportarem para você. Adquira o passe §l§aEMERALD§r §cor §l§bLEGEND§r §§mrank em §§abngmc.co/store §§mno site!",
+    "command.tphere.norequest": "Aquele jogador não enviou um pedido para que se teletransporte para eles.",
+    "command.tphere.receive": "{%sender} Pediu que se teleportasse para eles! Use §§a/tphere aceitar {%sender} para aceitar o pedido.",
```
[the remark this answers is not in the document]
Comment (hampuskraft) [reply portuguese.json]: Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).

File: portuguese.json [hunk 16]
```diff
@@ -0,0 +1,119 @@
...
+    "command.tphere.send": "O seu pedido de {%receiver} para se teletransportar para você foi enviado.",
+    "command.tpto.accepted.receiver": "Aceitou o pedido de {%sender} para que eles se teletransportem para você.",
```
[the remark this answers is not in the document]
Comment (hampuskraft) [reply portuguese.json]: Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).

File: portuguese.json [hunk 17]
```diff
@@ -0,0 +1,119 @@
...
+    "command.tpto.accepted.sender": "{%receiver} aceitou o seu pedido para que se teletransportasse para eles.",
```
[the remark this answers is not in the document]
Comment (hampuskraft) [reply portuguese.json]: Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).

File: portuguese.json [hunk 18]
```diff
@@ -0,0 +1,119 @@
...
+    "command.tpto.declined.receiver": "Recusou o pedido de {%sender} para que eles se teletransportassem para você.",
```
[the remark this answers is not in the document]
Comment (hampuskraft) [reply portuguese.json]: Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).

File: portuguese.json [hunk 19]
```diff
@@ -0,0 +1,119 @@
...
+    "command.tpto.declined.sender": "{%receiver} recusou o seu pedido de se teletransportar para eles.",
+    "command.tpto.noperm": "Não tem autorização para efetuar a teletransportação para outros jogadores. Adquira o passe §l§aEMERALD§r §§mor §l§bLEGEND§r §§mrank em  §§angmc.co/store §§mno site!",
```
[the remark this answers is not in the document]
Comment (hampuskraft) [reply portuguese.json]: Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).

File: portuguese.json [hunk 20]
```diff
@@ -0,0 +1,119 @@
...
+    "command.tpto.norequest": "Aquele jogador não enviou um pedido de teletransporte para você.",
+    "command.tpto.receive": "{%sender} pediu para se teletransportar para você! Use §§a/tpto accept {%0} para aceitar o pedido.",
+    "command.tpto.send": "O seu pedido de teletransporte para {%receiver} foi enviado.",
+    "command.vanish.disabled": "Agora você está visível!",
+    "command.vanish.enabled": "Agora você está invisivel",
```
[the remark this answers is not in the document]
Comment (hampuskraft) [reply portuguese.json]: Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).

File: portuguese.json [hunk 21]
```diff
@@ -0,0 +1,119 @@
...
+    "command.vote.slowdown": "Calma, já estamos verificando os votos para você!",
+    "db.error": "Ocorreu um erro inesperado na conexão com o banco de dados. Por favor, tente novamente mais tarde.",
+    "formhandler.reporter.notreported": "Esse jogador não pode ser reportado.",
+    "formhandler.reporter.reported": "Você reportou {%reported} por {%reason}",
+    "formhandler.reporter.staffmessage": "{%reported} foi reportado {%reporter} por {%reason}",
+    "formhandler.settings.disabled": "Desativado",
+    "formhandler.settings.enabled": "Ativado",
+    "formhandler.settings.fly": "{%enabledOrDisabled} voar no lobby.",
```
[the remark this answers is not in the document]
Comment (hampuskraft) [reply portuguese.json]: Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).

File: portuguese.json [hunk 22]
```diff
@@ -0,0 +1,119 @@
...
+    "formhandler.settings.fly.hidingrank": "Você não pode voar enquanto estiver usando um nome falso ou a esconder a sua posição de rank.",
```
[the remark this answers is not in the document]
Comment (hampuskraft) [reply portuguese.json]: Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).

File: portuguese.json (hunk 10 above)
[the remark this answers is not in the document]
Comment (hampuskraft) [reply portuguese.json]: Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).

File: portuguese.json (hunk 5 above)
[the remark this answers is not in the document]
Comment (hampuskraft) [reply portuguese.json]: Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).

File: portuguese.json [hunk 23]
```diff
@@ -0,0 +1,119 @@
...
+    "formhandler.settings.friends": "Agora vocês {%enabledOrDisabled} São amigos.",
+    "formhandler.settings.guildtag": "Agora é {%enabledOrDisabled} your guild tag.",
+    "formhandler.settings.hideplayers": "You are now {%enabledOrDisabled} players.",
```
[the remark this answers is not in the document]
Comment (hampuskraft) [reply portuguese.json]: Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).

File: portuguese.json [hunk 24]
```diff
@@ -0,0 +1,119 @@
...
+    "formhandler.settings.hiding": "escondido",
+    "formhandler.settings.knockback": "{%enabledOrDisabled} knockback no lobby.",
+    "formhandler.settings.announcements": "{%enabledOrDisabled} anúncios do servidor.",
+    "formhandler.settings.nick.enabled": "O sua nametag foi alterado para {%nametag}!",
+    "formhandler.settings.nick.reset": "Reajustando sua nametag.",
+    "formhandler.settings.pet.removed": "Pet removido.",
+    "formhandler.settings.pet.selected": "O seu pet de estimação foi mudado para um {%pet}.",
```
[the remark this answers is not in the document]
Comment (hampuskraft) [reply portuguese.json]: Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).

File: portuguese.json (hunk 6 above)
[the remark this answers is not in the document]
Comment (hampuskraft) [reply portuguese.json]: Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).

File: portuguese.json (hunk 1 above)
Comment (hampuskraft) [reply portuguese.json]: Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).

File: portuguese.json (hunk 2 above)
Comment (hampuskraft) [reply portuguese.json]: Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).

File: portuguese.json (hunk 4 above)
Comment (hampuskraft) [reply portuguese.json]: Resolved in [`3ea86c1` (#20)](https://github.com/NetherGamesMC/locale/pull/20/commits/3ea86c1437d4531c57431ef25b74aa5f3056cf10).

**Review by hampuskraft — APPROVED** [review]

**Comment by hampuskraft** [comment]
I handled all requested changes. Feel free to submit a PR if you would like to improve the fluency of something or other possible errors. Thanks!

**Comment by japakkjj** [comment]
Sorry for not answering, I was at the carnival, I saw you changed some things, could you chat with me at discord? Luk#8903
Enviado do meu iPhone

**Comment by hampuskraft** [comment]
If you would like to edit something, feel free to submit a new PR.

**Comment by japakkjj** [comment]
Ok, I'll send it to you and let you know, but you don't accept discord chat, do you?

Enviado do meu iPhone

> Em 21 de fev. de 2023, à(s) 13:23, Hampus Kraft ***@***.***> escreveu:
> 
> If you would like to edit something, feel free to submit a new PR.

**Comment by japakkjj** [comment]
Finished.

**Pull request merged**